packages/website/public/llms/styleGuide.md

```diff
@@ -0,0 +1,417 @@
+# Squiggle Style Guide
+
+## Limitations
+
+- There are floating point errors at high numbers (1e50 and above) and very small numbers (1e-10 and below). If you need to work with these, use logarithms if possible.
```

Comment:
It's a bit strange to have this on top when it's minor for most people, I'd move to the bottom and start with some other section.

Also, since it's not Squiggle-specific, it's just how IEEE 754 works, maybe would phrase it as "Squiggle doesn't have a bignum type" or something. Otherwise it sounds scarier than it is.

Reply:
My main concern was that the LLM knows - this was causing errors a few times. I'll add some detail and move down.